cmd/rdma/main.go

```diff
@@ -288,6 +281,30 @@ func (plugin *rdmaCniPlugin) CmdDel(args *skel.CmdArgs) error {
 	return nil
 }
 
+// getRDMADevice returns the first RDMA device found for the given deviceID.
+func (plugin *rdmaCniPlugin) getRDMADevice(deviceID string) (string, error) {
+	var rdmaDevs []string
+	if utils.IsPCIAddress(deviceID) {
+		rdmaDevs = plugin.rdmaManager.GetRdmaDevsForPciDev(deviceID)
+		if len(rdmaDevs) == 0 {
+			return "", errors.New("no RDMA devices found")
```

Comment:
nit: add deviceId to the error string. It's better to have redundant information when it comes to bugs.

Same thing on L295